gcs/src/dashboard.jsx

```diff
@@ -123,6 +159,27 @@ export default function Dashboard() {
   const [playArmed] = useSound(armSound, { volume: 0.1 })
   const [playDisarmed] = useSound(disarmSound, { volume: 0.1 })
 
+  // Incoming messages
+  const [possibleData, setPossibleData] = useState({})
+  const [wantedData, setWantedData] = useState({})
+
+  // Data Modal
+  const [opened, { open, close }] = useDisclosure(false)
+  // Test Message Data 
+  const testData = {
+    "airspeed": 1.6184577941894531,
+    "groundspeed": 0.0274711474776268,
+    "heading": 227,
+    "throttle": 0,
+    "alt": 145.0500030517578,
+    "climb": 0.004107808228582144,
+    "timestamp": 1720022850.1995258
+  };
```

Comment:
This shouldn't be in here right cuz it's test data?

Reply:
Yeah, I'll clear it.